esp_encrypted_img/pre_encrypted_ota/README.md

```diff
@@ -0,0 +1,54 @@
+| Supported Targets | ESP32 | ESP32-C2 | ESP32-C3 | ESP32-C5 | ESP32-C6 | ESP32-C61 | ESP32-P4 | ESP32-S2 | ESP32-S3 |
```

Comment:
The example should be located in `esp_encrypted_img/examples/pre_encrypted_ota`.